fares-data-build-tool/fdbt-site
# Retain user input on page with conditionally-revealing inputs

# Description

-   This PR allows for the user input within conditional input elements to be retained when the user is redirected back to the page (due to the error state, clicking a confirmation page link, or by manually redirecting back to that page).
-   Pages with conditional inputs are: /definePassengerType, /defineTimeRestrictions, /periodValidity, /returnValidity, /productDateInformation.

# Testing instructions

-   Pull down this branch and run the site locally. Validate that all pages with conditional input components work as expected.
-   Check that each of /definePassengerType, /defineTimeRestrictions and /periodValidity now retain any user input.

# Type of change


# Checklist:
